kubernetes/resource_kubernetes_default_service_account.go

```diff
@@ -59,6 +60,25 @@ func resourceKubernetesDefaultServiceAccountCreate(ctx context.Context, d *schem
 		return diag.FromErr(err)
 	}
 
+	if v, ok := d.GetOkExists("automount_service_account_token"); ok {
+		ops := patchMetadata("metadata.0.", "/metadata/", d)
+		ops = append(ops, &ReplaceOperation{
+			Path:  "/automountServiceAccountToken",
+			Value: v.(bool),
+		})
```

Comment:
Could you please also add the replacement operation for the `image_pull_secret` and `secret` arguments?